src/enkf/gridio_fv3reg.f90

```diff
@@ -476,6 +492,27 @@ subroutine readgriddata(nanal1,nanal2,vars3d,vars2d,n3d,n2d,levels,ndim,ntimes,
 
            endif
            
+           if (dbz_ind > 0) then
+               varstrname = 'ref_f3d'
+               call fv3lamfile%get_idfn(varstrname,file_id,fv3filename)
+               call read_fv3_restart_data3d(varstrname,fv3filename,file_id,workvar3d)
+               do k=1,nlevs
+                   nn = nn_tile0
+                   do j=1,ny_res
+                     do i=1,nx_res
+                       nn=nn+1
+                       vargrid(nn,levels(dbz_ind-1)+k,nb,ne)=max(workvar3d(i,j,nlevs+1-k),0.0_r_kind)
+                     enddo
+                   enddo
+               enddo
+               do k = levels(dbz_ind-1)+1, levels(dbz_ind)
+                   if (nproc .eq. 0)                                               &
+                      write(6,*) 'READFVregional : qnr ',                           &
```

Comment:
@chunhuazhou @hu5970 Changing "qnr" to "dbz"?

Reply:
Done